lib/ansible/modules/command.py

```diff
@@ -52,6 +52,11 @@
     description:
       - A filename or (since 2.0) glob pattern. If a matching file exists, this step B(will) be run.
     version_added: "0.8"
+  changes:
+    type: path
+    description:
+      - A filename or (since 2.0) glob pattern. If a matching file exists, this step B(will) be run.
```

Comment:
Since this is a new feature, the "(since 2.0)" statement can be removed. The description is incorrect. This only controls task change reporting, not whether or not the command will be run. The command will always run (except in check mode), but the task will only report changed if the file changed. This is an import difference in behavior from `creates` and `removes` that should be explained.
```suggestion
      - Path to a file or glob pattern. If a match exists and it is a file, the command will B(always) be run except in check mode. If any files matching the path or pattern are changed after running the command, the task will report a change. 
```